cmd/modelcmd/modelcommand.go

```diff
@@ -133,12 +135,11 @@ func (c *ModelCommandBase) maybeInitModel() error {
 	// returned [ErrNoModelSpecified,ErrNoControllersDefined,ErrNoCurrentController]
 	// at that point and so need to try again.
 	// If any other error result was returned, we bail early here.
-	retriableError := func(original error) bool {
-		return errors.Cause(c.initModelError) != ErrNoModelSpecified &&
-			errors.Cause(c.initModelError) != ErrNoControllersDefined &&
-			errors.Cause(c.initModelError) != ErrNoCurrentController
+	noRetry := func(original error) bool {
```

Comment:
I assume this is fixing the name because this is testing whether we *shouldn't* retry, rather than we *should* ?

I do sometimes miss pythons "in" syntax for stuff like this:

eg if (c in [ErrNoModelSpecified, ErrNoControllersDefined, ErrNoCurrentController]):

creating a map to do an 'ok' check just isn't as satisfying :)

Reply:
Correct; the name was the inverse of the intent.

I wax and wane on the idea, but I do think some more wrist friendly collection syntax additions would make Go better without compromising on the "don't hide complexity behind syntactic sugar" philosophy.